feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt

```diff
@@ -143,6 +145,10 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
                 binding.isCollapsed = isCollapsed
             }
         }
+        sessionPagesStore.onGoingSessionOnTabReselected(args.day).nonNull()
+            .observe(viewLifecycleOwner) {
+                binding.sessionsRecycler.scrollToPosition(it)
```

Comment:
I think it is better to observe `sessionPagesStore.reselectedTab`. And create a method like `scrollToCurrentSession()`.
I think `onGoingSessionOnTabReselected` is a little hard 💭

Reply:
That's certainly...
Thanks, I fix it.

Reply:
Probably you should write `if statement` in observe callback method.
And can you use SessionPage for SessionPageReselected?

Reply:
Uhmm...
In OnTabSelectedListener, nullable cast to `SessionPage.Day` is done.
Althought it is not type-safe certainly, I wanted the Day#dayNumber, so I had used `day int`...

I try using session.

Reply:
Well.. Can you use `SessionPage.pageOfDay(args.day)`?